codegen/templates/generated.gotpl

```diff
@@ -204,30 +205,36 @@ func (ec *executionContext) FieldMiddleware(ctx context.Context, obj interface{}
 	}()
 	{{- if .Directives }}
 	rctx := graphql.GetResolverContext(ctx)
-	for _, d := range rctx.Field.Definition.Directives {
-		switch d.Name {
-		{{- range $directive := .Directives }}
-		case "{{$directive.Name}}":
-			if ec.directives.{{$directive.Name|ucFirst}} != nil {
-				{{- if $directive.Args }}
-					rawArgs := d.ArgumentMap(ec.Variables)
-					{{ template "args.gotpl" $directive.Args }}
-				{{- end }}
-				n := next
-				next = func(ctx context.Context) (interface{}, error) {
-					return ec.directives.{{$directive.Name|ucFirst}}({{$directive.CallArgs}})
+	res := func() interface{} {
```

Comment:
Why was this required?

Reply:
I changed to return values 1 to 2. but generated code will return `return graphql.Null`.
this immidiately execute function is wrapping it.